<launch>
    <param name="logging_enabled" value="false"/>
    <param name="log_folder" value="$(find smmap)/../logs/"/>

    <param name="deformable_type" value="rope"/>
    <param name="task_type" value="coverage"/>
</launch>
